indexer/editable_map_object.cpp

```diff
@@ -94,12 +96,34 @@ void EditableMapObject::SetNearbyStreets(vector<string> && streets)
 {
   m_nearbyStreets = move(streets);
 }
-void EditableMapObject::SetHouseNumber(string const & houseNumber)
+
+bool EditableMapObject::SetHouseNumber(string const & houseNumber)
 {
-  // TODO(AlexZ): Check house number for validity with feature::IsHouseNumber ?
+  // TODO: Improve this basic limit.
+  auto constexpr kMaxHouseNumberLength = 25;
```

Comment:
Нельзя так просто return, надо очистить поле дома в этом объекте.

> On Apr 6, 2016, at 19:42, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In indexer/editable_map_object.cpp <https://github.com/mapsme/omim/pull/2802#discussion_r58739682>:
> 
> >  {
> > -  // TODO(AlexZ): Check house number for validity with feature::IsHouseNumber ?
> > +  // TODO: Improve this basic limit.
> > +  auto constexpr kMaxHouseNumberLength = 25;
> if (houseNumber.empty())
>   return true;
> ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2802/files/28c906af0d4709e796dd4fb05bf395153131032c#r58739682>